pkg/yurthub/storage/disk/storage.go

```diff
@@ -42,9 +41,6 @@ type diskStorage struct {
 
 // NewDiskStorage creates a storage.Store for caching data into local disk
 func NewDiskStorage(dir string) (storage.Store, error) {
-	if dir == "" {
```

Comment:
It's need to check the validity of input parameter. so please do not delete it.

Reply:
Actually, nil check is not enough for validity if we allow user input directory. We need to check more such as the directory is at least available.

Reply:
> Actually, nil check is not enough for validity if we allow user input directory. We need to check more such as the directory is at least available.

@Fei-Guo The following code can cover it what the `dir` is empty or not valid path.

Reply:
> It's need to check the validity of input parameter. so please do not delete it.

I have fixed it